### ISSUE TITLE
Adding dask to Code Library Search plugin

### ISSUE THREAD

**Issue by mswoff** [dask/community#332]
I wanted to let you know that users of our Code Library Search Chatgpt plugin have requested we support dask. Unless we hear otherwise from you, we’ll add support for dask (free for open-source projects and no action required from your team).

The plugin let's users get library specific coding help by analyzing your library's codebase.

If you want to get in touch about any concerns or offering AI based support directly on your forum/website, please email me at mason@salv.ai.

**Comment by fjetter** [comment]
Thanks for the headsup @mswoff . I transferred this ticket to the community tracker which feels more appropriate

cc @jrbourbeau @jacobtomlinson @mrocklin @quasiben

**Comment by jacobtomlinson** [comment]
No immediate concerns from me.

**Comment by jrbourbeau** [comment]
Yeah, no concerns from me either. It sounds like there's nothing to do on our end -- is that correct @mswoff?
